staging/src/k8s.io/apiserver/pkg/server/healthz/healthz.go

```diff
@@ -160,12 +161,13 @@ func InstallLivezHandler(mux mux, checks ...HealthChecker) {
 // InstallPathHandler more than once for the same path and mux will
 // result in a panic.
 func InstallPathHandler(mux mux, path string, checks ...HealthChecker) {
-	InstallPathHandlerWithHealthyFunc(mux, path, nil, checks...)
+	InstallPathHandlerWithCustomisation(mux, path, nil, false, checks...)
 }
 
-// InstallPathHandlerWithHealthyFunc is like InstallPathHandler, but calls firstTimeHealthy exactly once
-// when the handler succeeds for the first time.
-func InstallPathHandlerWithHealthyFunc(mux mux, path string, firstTimeHealthy func(), checks ...HealthChecker) {
+// InstallPathHandlerWithHealthyFunc is like InstallPathHandler but:
+// - calls firstTimeHealthy exactly once when the handler succeeds for the first time
+// - allows for disabling putting a stacktrace for httplog for the current request so that the output is condensed
+func InstallPathHandlerWithCustomisation(mux mux, path string, firstTimeHealthy func(), disableStacktraceForHttpLog bool, checks ...HealthChecker) {
```

Comment:
let's not change the name at all. It keeps the carry change small, and I think there is not much value in `WithCustomization`.

Reply:
done.